<!-- obstacle_detector launch file -->

<launch>

	<group ns="obstacle_detector">
		<node pkg="obstacle_detector" name="obstacle_detector" type="obstacle_detector.py">
			<param name="threshold_slow" value="1.0" />
			<param name="threshold_stop" value="0.3" />
			<param name="threshold_ignore" value="0.05" />
			<param name="laser_scan" value="/scan" />
		</node>
	</group>

	<node name="sick_tim310s01" pkg="sick_tim" type="sick_tim310s01" respawn="false" output="screen">
	    <param name="min_ang" type="double" value="-2.35619449019" />
	    <param name="max_ang" type="double" value="2.35619449019" />
	    <param name="intensity" type="bool" value="True" />
	    <param name="skip" type="int" value="0" />
	    <param name="frame_id" type="str" value="laser" />
	    <param name="time_offset" type="double" value="-0.001" />
	    <param name="publish_datagram" type="bool" value="False" />
	</node>

</launch>

